FirebaseMLModelDownloader/Tests/Integration/ModelDownloaderIntegrationTests.swift

```diff
@@ -362,4 +362,57 @@ final class ModelDownloaderIntegrationTests: XCTestCase {
       }
     }
   }
+
+  /// Test logging telemetry event.
+  func testLogTelemetryEvent() {
+    guard let testApp = FirebaseApp.app() else {
+      XCTFail("Default app was not configured.")
+      return
+    }
+
+    let testModelName = "image-classification"
+    let testName = "model-downloader-test"
+
+    let conditions = ModelDownloadConditions()
+    let modelDownloader = ModelDownloader.modelDownloaderWithDefaults(
+      .createTestInstance(testName: testName),
+      app: testApp
+    )
+
+    /// Test download type - latest model.
+    let downloadType: ModelDownloadType = .latestModel
+    let latestModelExpectation = expectation(description: "Get latest model.")
+
+    modelDownloader.getModel(
+      name: testModelName,
+      downloadType: downloadType,
+      conditions: conditions,
+      progressHandler: { progress in
+        XCTAssertLessThanOrEqual(progress, 1)
+        XCTAssertGreaterThanOrEqual(progress, 0)
+      }
+    ) { result in
+      switch result {
+      case let .success(model):
+        guard let telemetryLogger = TelemetryLogger(
+          isStatsEnabled: true,
+          apiKey: testApp.options.apiKey,
+          projectID: testApp.options.projectID
+        ) else {
+          XCTFail("Could not initialize logger.")
+          return
+        }
+        telemetryLogger.logModelDownloadEvent(
```

Comment:
I assume it is a temporary code for testing/debugging, correct? If yes, could you please add a TODO. If no, could you please explain what's the purpose of calling telemetryLogger here.

Reply:
Yep good catch, this was for testing. I'll remove it before we launch.